nixos/tests/multipass.nix

```diff
@@ -0,0 +1,39 @@
+import ./make-test-python.nix ({ pkgs, lib, ... }:
+
+let
+  multipass-image = import ../release.nix {
+    configuration = {
+      # Building documentation makes the test unnecessarily take a longer time:
+      documentation.enable = lib.mkForce false;
+    };
+  };
+
+in
+{
+  name = "multipass";
+
+  meta = with pkgs.lib.maintainers; {
+    maintainers = [ jnsgruk ];
+  };
```

Comment:
```suggestion
  meta.maintainers = [ lib.maintainers.jnsgruk ];
```

Reply:
Fixed! Thanks again for all the feedback!

Reply:
Fixed! Thanks again for all the feedback!